prov/tcp/src/tcpx_ep.c

```diff
@@ -61,6 +63,11 @@ void tcpx_hdr_bswap(struct tcpx_base_hdr *hdr)
 		ptr += sizeof(uint64_t);
 	}
 
+	if (hdr->flags & TCPX_TAGGED) {
+		*((uint64_t *)ptr) = ntohll(*((uint64_t *) ptr));
```

Comment:
can TCPX_REMOTE_CQ_DATA and TCPX_TAGGED both be set?  If so, could this be reversing the above ntohll swap?  if not, should this be an `else if` to make it obvious that both can't be true?

Reply:
Both flags can be set.  ptr is incremented after swapping the cq data.  Same after swapping the tag.  It's the current location in the header being processed.

Reply:
got it. thanks!